### PR TITLE
Added contextual menu to stories in dashboard

### PR DESCRIPTION
**Pull request by carloskelly13**
https://github.com/google/web-stories-wp/issues/767

- Adds contextual menu support plus dividers to the dashboard
- Updates storybook with the new context-menu grid
- Position of menu and actions will be in future PRs

Kapture 2020-04-07 at 11 44 20

### CODE REVIEW

**Review by mariano-formidable — CHANGES_REQUESTED** [review]
Everything looks good, but just one small nit.  Shouldn't the options menu have a pointer cursor?
image

**Comment by carloskelly13** [comment]
@mariano-formidable Good call! Added that in!

**Review by mariano-formidable — APPROVED** [review]
Looks good to me!

File: assets/src/dashboard/components/cardGridItem/cardItemMenu.js [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+        framelessButton
+        onSelect={onMenuItemSelected}
+        items={STORY_CONTEXT_MENU_ITEMS}
+      />
+    </MenuContainer>
+  );
+}
+
+CardItemMenu.propTypes = {
+  story: StoryPropType,
```
Comment (BrittanyIRL) [inline assets/src/dashboard/components/cardGridItem/cardItemMenu.js:91]: it looks like the only part of the `story` getting used is the `id`. Thoughts on passing just the id in?

File: assets/src/dashboard/components/popover-menu/index.js [hunk 2]
```diff
@@ -81,6 +81,12 @@ const MenuItemContent = styled.span`
   margin: auto 0;
 `;
 
+const Separator = styled.li`
```
Comment (BrittanyIRL) [inline assets/src/dashboard/components/popover-menu/index.js:84]: nit: feels like this should be a span not a list item

File: assets/src/dashboard/components/popover-menu/index.js [hunk 3]
```diff
@@ -140,20 +146,27 @@ const PopoverMenu = ({
...
-      >
-        <MenuItemContent>{item.label}</MenuItemContent>
-      </MenuItem>
-    );
-  };
+  const renderMenuItem = useCallback(
+    (item, index) => {
+      const itemIsDisabled = !item.value && item.value !== 0;
+      return (
+        <MenuItem
```
Comment (BrittanyIRL) [inline assets/src/dashboard/components/popover-menu/index.js:153]: would an alternative to the li/span break be checking for a separator and passing it as a prop to the menuItem and rendering a border?

**Review by BrittanyIRL — APPROVED** [review]
I was too slow! Here's my review anyways. All nits, nothing that would have blocked a merge.

**Pull request merged**